develop/api_metadata.rst

```diff
@@ -366,3 +333,231 @@ Caveats:
+
+Upon beginning a connection, a node consults the table to see whether a row with the destination ``nodeid`` and desired ``rolename`` exists. If so, the node includes the corresponding ``authinfo`` string in its libpq connection. A common example is to store a password, like ``'password=abc123'``, but you can review the `full list <https://www.postgresql.org/docs/current/static/libpq-connect.html#LIBPQ-PARAMKEYWORDS>`_ of possibilities.
+
+The parameters in ``authinfo`` are space-separated, in the form ``key=val``. To write an empty value, or a value containing spaces, surround it with single quotes, e.g., ``keyword='a value'``. Single quotes and backslashes within the value must be escaped with a backslash, i.e., ``\'`` and ``\\``.
+
+The ``nodeid`` column can also take the special values 0 and -1, which mean *all nodes* or *loopback connections*, respectively. If, for a given node, both specific and all-node rules exist, the specific rule has precedence.
+
+::
+
+    SELECT * FROM pg_dist_authinfo;
+
+     nodeid | rolename | authinfo
+    --------+----------+-----------------
+        123 | jdoe     | password=abc123
+    (1 row)
+
+Connection Pooling Credentials
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+  This table is a part of Citus Enterprise Edition. Please `contact us <https://www.citusdata.com/about/contact_us>`_ to obtain this functionality.
+
+The ``pg_dist_poolinfo`` table holds the host, port and database name for Citus to use when connecting to a node through a connection pooler. This table is specific to connection pooling rather than direct connections between nodes, and overrides the values in :ref:`pg_dist_node <pg_dist_node>`.
+
++----------+---------+---------------------------------------------------+
+| Name     | Type    | Description                                       |
++==========+=========+===================================================+
+| nodeid   | integer | Node id from :ref:`pg_dist_node`                  |
++----------+---------+---------------------------------------------------+
+| poolinfo | text    | Space-separated parameters: host, port, or dbname |
++----------+---------+---------------------------------------------------+
+
+.. code-block:: sql
+
+   -- how to connect to node 1 (as identified in pg_dist_node)
+
+   INSERT INTO pg_dist_poolinfo (nodeid, poolinfo)
+        VALUES (1, 'host=127.0.0.1 port=5433');
+
+.. note::
+
+   :ref:`Shard rebalancing <shard_rebalancing>` is not compatible with connection poolers such as pgbouncer and ignores the settings in ``pg_dist_poolinfo``.
+
+.. _worker_shards:
+
+Shards and Indices on Workers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Worker nodes store shards as tables that are ordinarily hidden (see :ref:`override_table_visibility`). The easiest way to obtain information about the shards on each worker is to consult that worker's ``citus_shards_on_worker`` view. For instance, here are some shards on a worker for the distributed table ``test_table``:
```

Comment:
Does this view also exist on the coordinator?

Reply:
It does, and is empty.

However... Now that I test it, it's empty on the workers too even with a table distributed. Asked people to check this out in slack.

Reply:
Please see my comment regarding this in the above comments.